sources/models/src/modeled_types/oci_defaults.rs

```diff
@@ -125,5 +155,11 @@ mod oci_defaults_rlimits {
             "max-open-files",
             "RLIMIT_NOFILE",
         );
+
+        check_rlimit_strings(
+            OciDefaultsResourceLimitType::MaxLockedMemory,
+            "max-locked-memory",
+            "RLIMIT_MEMLOCK",
+        );
```

Comment:
Is there a reason there is only this one new test and not one for all of the new settings we are adding? What is special about this setting?

Reply:
There is no strong reason for testing only this one but as this test is to check the mapping so I thought it will be redundant to test for all the settings.

Reply:
I find it a little odd to have only 2 unit tests for two of the sixteen settings we have here. Either testing that any one comes through correctly as a basic test makes sense or testing that each setting ends up where we want it makes sense, but just these two are hard to reason about. Its a lot of extra boilerplate but having one test for each setting is my preference since it would cover each case.

Reply:
Sure Its worth to have checks more always.. I will add one test for all 16 settings